tests/foreman/api/test_syncplan.py

```diff
@@ -4,11 +4,16 @@
 from fauxfactory import gen_string
 from robottelo import entities
 from robottelo.common.decorators import (
-    data, run_only_on, stubbed)
+    data,
+    run_only_on,
+    skip_if_bug_open,
+    stubbed,
+)
 from robottelo.test import APITestCase
 # (too-many-public-methods) pylint:disable=R0904
 
 
+@stubbed
```

Comment:
> Try running with py.test or plain unittest and you will see an exception raised, Nose masks that exception.

Ouch! I didn't realize that an exception was being raised and hidden. Thanks for the heads up.